src/app/molecules/message/Message.jsx

```diff
@@ -174,11 +174,34 @@ const MessageBody = React.memo(({
 
   const content = isCustomHTML
     ? twemojify(sanitizeCustomHtml(body), undefined, true, false)
-    : <p>{twemojify(body, undefined, true)}</p>;
+    : twemojify(body, undefined, true);
```

Comment:
the `<p>` tag was used to apply the `white-space: pre-wrap`  see: (https://github.com/ajbura/cinny/blob/9854f4eb2d4b7fb90f063a29703b8a348de0aafb/src/app/molecules/message/Message.scss#L160)

also it is not used with customHtml is because it wraps the text in appropriate tags automatically.

Reply:
Gotcha!  I'll figure out how to work around that.